### ISSUE TITLE
[CI] Cannot import name 'docevents' from boto3

### ISSUE THREAD

**Issue by barry-jin** [dmlc/gluon-nlp#1369]
## Description
Docevents was removed in botocore 1.18.0, which caused our workflow broken. Should we freeze boto3 version to stable version 1.14.32   boto/boto3#2596

### Error Message
See [artifacts](https://github.com/dmlc/gluon-nlp/actions/runs/265742028) in most recent CI action. 
Snippet of AWS batch log stream. 
[2020-09-21T20:31:10.865Z] Traceback (most recent call last):
[2020-09-21T20:31:10.865Z]   File "/usr/local/bin/aws", line 19, in <module>
[2020-09-21T20:31:10.865Z]     import awscli.clidriver
[2020-09-21T20:31:10.865Z]   File "/usr/local/lib/python3.6/dist-packages/awscli/clidriver.py", line 36, in <module>
[2020-09-21T20:31:10.865Z]     from awscli.help import ProviderHelpCommand
[2020-09-21T20:31:10.865Z]   File "/usr/local/lib/python3.6/dist-packages/awscli/help.py", line 23, in <module>
[2020-09-21T20:31:10.865Z]     from botocore.docs.bcdoc import docevents
[2020-09-21T20:31:10.865Z] ImportError: cannot import name 'docevents'

## To Reproduce
Set boto3==1.14.32 in [setup.py](https://github.com/dmlc/gluon-nlp/blob/master/setup.py)

### Steps to reproduce
(Paste the commands you ran that produced the error.)

1.
2.

## What have you tried to solve it?

1.
2.

## Environment

We recommend using our script for collecting the diagnositc information. Run the following command and paste the outputs below:
```
curl --retry 10 -s https://raw.githubusercontent.com/dmlc/gluon-nlp/master/tools/diagnose.py | python

# paste outputs here
```

**Comment by sxjscience** [comment]
Would you submit a PR to fix this?

**Comment by barry-jin** [comment]
closes via #1370
